source/catalog.html

```diff
@@ -225,38 +225,61 @@ <h2 class="visually-hidden">Запостить своё фото.</h2>
         <form class="upload-photo__form" method="post" action="https://echo.htmlacademy.ru">
           <img class="upload-photo__img" src="https://via.placeholder.com/280x374" width="280" height="374" alt="Фото для загрузки. Зимняя дорога под снегом в ясный день.">
           <div class="upload-photo__editing-container">
-            <ul class="upload-photo__editing editing-tools">
-              <li class="editing-tools__item">
-                <button class="editing-tools__button editing-tools__button--crop">
+            <ul class="upload-photo__editing-controls editing-controls-list">
+              <li class="editing-controls-list__item">
+                <button type="button" class="editing-controls-list__button editing-controls-list__button--crop">
                   <span class="visually-hidden">Кадрирование.</span>
+                  <svg class="editing-controls__icon" width="44" height="44" viewBox="0 0 44 44" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
```suggestion
                  <svg class="editing-controls__icon" width="44" height="44" viewBox="0 0 44 44" fill="none">
```

Можно удалить `xmlns="http://www.w3.org/2000/svg"`, он нужен только в файле, так браузер распознает формат файла. А когда встраиваешь код, то браузер и так распознает формат svg =)